start_marple_marvin.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+source /projects/wp3/nobackup/TwistCancer/Bin/venv_marple/bin/activate
+pip install -r /projects/wp3/nobackup/TwistCancer/Bin/marple_rd_tc/requirements.txt
+module load slurm-drmaa/1.1.3
+
+## To match stackstorm outline
+old_varible=$1
+sequenceid=$2
```

Comment:
The old_varible is not used in the script from what I see. Shouldn't sequenceid be $1 in that case?